encoding/masking-limiting-etc.md

```diff
@@ -83,22 +126,107 @@ Internally it uses a mask called a Camembert to generate a larger mask
 and limits it to the area affected by a line-darkening script.
 The main mask has no name and is simply dhhmask(mode=3).
 
-Example: Build a simple dehalo mask
+For more information about edgemasks, see [here](https://kageru.moe/blog/article/edgemasks).
```

Comment:
It's almost never wise to use `here` as the link text. The actual link should also be specified after the paragraph as in-text linking is reserved for links within the same page.